tests/test_synthetic_data.py

```diff
@@ -27,7 +27,8 @@ def _calc_shift(dx_, vel):
     i_shift = _calc_shift(dx_grid, u)
     j_shift = _calc_shift(dy_grid, v)
     ds_new = ds_scalars.roll(dict(x=i_shift, y=j_shift), roll_coords=False)
-    ds_new["time"] = ds_scalars.time + np.timedelta64(int(dt), "s")
+
+    ds_new = ds_new.assign_coords(time=ds_scalars.time + np.timedelta64(int(dt), "s"))
```

Comment:
good catch! It was a bit of a pain when xarray deprecated what I had before, but it is better to by explicit like this